internal/app/api/v1/server.go

```diff
@@ -45,7 +45,10 @@ import (
 	"github.com/kubeshop/testkube/pkg/utils/text"
 )
 
-const HeartbeatInterval = time.Hour
+const (
+	HeartbeatInterval = time.Hour
+	HttpBodyLimit     = 1 * 1024 * 1024 * 1024 // 1GB - needed for file uploads
```

Comment:
need to have helm value for this

Reply:
I will create a follow-up ticket, just want to fix the issue of the user for now